definitions/infra-kubernetes_statefulset/definition.yml

```diff
@@ -0,0 +1,25 @@
+type: KUBERNETES_STATEFULSET
```

Comment:
Missing `domain` as the bot said. The other errors are not yours 😅 we're fixing them now.

Reply:
Already force-pushed adding the `domain` 😄